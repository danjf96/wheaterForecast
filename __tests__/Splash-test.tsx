/**
 * @format
 */

 import 'react-native';
 import React from 'react';
 import Splash from '../src/screens/splash'
 // Note: test renderer must be required after react-native.
 import { render } from 'react-native-testing-library';
 
 it('renders correctly', async () => {
   const screen = render(<Splash />)
   expect(screen.toJSON()).toMatchSnapshot()
 });
 