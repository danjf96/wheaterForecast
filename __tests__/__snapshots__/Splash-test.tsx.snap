// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0086ff",
      "flexDirection": "column",
      "height": "100%",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Image
    collapsable={false}
    forwardedRef={[Function]}
    source={
      Object {
        "testUri": "../../../src/assets/images/logo.png",
      }
    }
    style={
      Object {
        "opacity": 0,
        "resizeMode": "contain",
        "transform": Array [
          Object {
            "translateX": 200,
          },
          Object {
            "skewX": "-30deg",
          },
        ],
        "width": "90%",
      }
    }
    testID="animation"
  />
</View>
`;
